vendor/turf/Makefile

```diff
@@ -0,0 +1,12 @@
+# Package details
+export VENDOR ?= cloudposse
+export APK_BUILD_TEMPLATE ?= APKBUILD.github-binary
+export DOWNLOAD_URL ?= $(PACKAGE_REPO_URL)/releases/download/$(PACKAGE_VERSION)/$(PACKAGE_NAME)_$(OS)_$(ARCH)
+
+include ../../tasks/Makefile.vendor_includes
+
+install:
+	$(call download_binary)
+
+test:
+	$(PACKAGE_EXE) --version
```

Comment:
https://github.com/cloudposse/packages/blob/master/vendor/terraform/Makefile

Update you test like this so we check version

Reply:
b11331689c4cbbf811e539c6d6b8015ba804e9d7